### ISSUE TITLE
Reword stacktrace language to avoid ambiguity

### ISSUE THREAD

**Issue by willow-ahrens** [JuliaLang/julia#50112]
When Julia prints a stacktrace, a message e.g. "the last 3 lines are repeated 12 times" sometimes gets printed. This is confusing to me because I'm sometimes not sure which lines are referred to by "last" (stacktraces have bidirectional interpretations. Temporally, the last 3 frames might be understood as parent frames). Can we reword to "above", e.g. "the above 3 lines are repeated 12 times"? I've included a simple unambiguous example.

```
julia> foo(n) = bar(n + 1)
foo (generic function with 1 method)

julia> bar(n) = n < 100 ? foo(n + 1) : error()
bar (generic function with 1 method)

julia> foo(0)
ERROR: 
Stacktrace:
   [1] error()
     @ Base ./error.jl:44
   [2] bar(n::Int64)
     @ Main ./REPL[9]:1
   [3] foo(n::Int64)
     @ Main ./REPL[8]:1--- the last 2 lines are repeated 50 more times ---
```

**Comment by inkydragon** [comment]
Related code:
https://github.com/JuliaLang/julia/blob/a31c8ef47d446f071a7c48f332733e67e32a4dfe/base/errorshow.jl#L666-L668

**Comment by hsradA23** [comment]
I've submitted a PR with the suggested changes.
#50131

**Comment by vtjnash** [comment]
Fixed by #50133
